tests/cases/fourslash/formattingOnSingleLineBlocks.ts

```diff
@@ -1,16 +1,11 @@
 /// <reference path='fourslash.ts' />
 
-/////*1*/class C
-////{}/*2*/
-/////*3*/if (true)
-////{}/*4*/
+////class C

```

Comment:
Would it be interesting to test the other places braces are allowed?  e.g. object literals, lambda bodies, etc?